lib/watir/locators/element/locator.rb

```diff
@@ -44,10 +44,8 @@ def locate
                       find_first_by_multiple
                     end
 
-          # This actually only applies when finding by xpath/css - browser.text_field(:xpath, "//input[@type='radio']")
-          # We don't need to validate the element if we built the xpath ourselves.
-          # It is also used to alter behavior of methods locating more than one type of element
-          # (e.g. text_field locates both input and textarea)
+          # Validation not necessary if watir builds the xpath
+          return element unless @selector.key?(:xpath) || @selector.key?(:css)
```

Comment:
Ugh, except this won't work without also updating [this line](https://github.com/p0deje/watizzle/blob/master/lib/watizzle/locators/element/locator.rb#L33) since we also want to validate `:id` parameters, but we can't check it in `Validator#validate` because we delete the key. :-/

I think the subclasses of `Locator#locate` will need to be updated for this to work.